integration/test/performance_tracker/transmitter/transmitter.go

```diff
@@ -0,0 +1,237 @@
+package transmitter
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+)
+const(
+	TIME_CONSTANT = 300.0 // this const is in seconds , 5 mins
+)
+type TransmitterAPI struct{
+	DynamoDbClient dynamodbiface.DynamoDBAPI
+	DataBaseName      string // this is the name of the table when test is run
+}
+// this is the packet that will be sent converted to DynamoItem
+type Metric struct{
+	Average float64
+	StandardDev float64
+	Period	int 	//in seconds
+	Data []float64
+}
+
+type collectorData []struct{ // this is the struct data collector passes in
+	Id string	`json:"Id"`
+	Label string	`json:Label`	
+	Messages string	`json:Messages`
+	StatusCode string	`json:StatusCode`
+	Timestamps []string	`json:Timestamps`
+	Values []float64	`json:Values`
+}
+
+
+/*
+InitializeTransmitterAPI
+Desc: Initializes the transmitter class
+Side effects: Can create a dynamodb table
+*/
+func InitializeTransmitterAPI(DataBaseName string) * TransmitterAPI{
+	transmitter := new(TransmitterAPI)
+	//setup aws session
+	session := session.Must(session.NewSessionWithOptions(session.Options{
+		SharedConfigState: session.SharedConfigEnable,
+	}))
+	transmitter.DynamoDbClient =  dynamodb.New(session)
+	transmitter.DataBaseName = DataBaseName //fmt.Sprintf("%d",int(time.Now().UnixNano()))
+	// check if the dynamo table exist if not create it
+	tableExist, err:= transmitter.TableExist()
+	if err !=nil{
+		return nil
+	}
+	if !tableExist{
+		fmt.Println("Table doesn't exist")
+		err := transmitter.CreateTable()
+		if err != nil{
+			fmt.Println("Couldn't create table")
```

Comment:
Isn't this print statement redundant? In the `CreateTable()` function, it prints out any error that occurs when trying to create the table. This just doubles up on logging and doesn't provide any additional context or value.